### DISCUSSION TITLE
Japanese input is very buggy in the new Issues/Projects

### DISCUSSION THREAD

**Discussion by mesqueeb** [community/community#6358] [Projects and Issues] [answered]
When trying to write anything in Japanese in the new `beta` Issues/Projects, it will submit a new task EVERY TIME I choose a character from the conversion table...

Please only submit tasks on keycode 13, and not on the Enter key when choosing a kanji (which is not 13 anyway)...

In its current state typing in Japanese / Chinese is impossible.

GIF: 
![2021-10-15 16 56 21](https://user-images.githubusercontent.com/3253920/137452746-84bab37c-6989-495d-8433-d1f55db6ffa8.gif)

**Comment by peaceiris** [comment]
Nice catch! Upvoted.

**Comment by RickCogley** [comment]
I'm not seeing this, and Japanese entry is working for me. My setup might be weird tho. I have an English keyboard macbook pro, macOS 11.6 Big Sur latest, and I'm entering Japanese using ATOK Passport, typing in romaji, selecting with arrow keys, and pressing return to select. 

Edit: tried using the standard Mac IME for this and I still have it working... wonder what's different about my setup...

**Reply by mesqueeb** [reply]
@RickCogley my setup is MacOS 11.6 with Safari Browser and Google Japanese input: https://www.google.co.jp/ime/

**Reply by RickCogley** [reply]
ah, right, the browser I tested on is Firefox 92.0.1 (64-bit). FYI

**Comment by peaceiris** [comment]
I am facing this.

Only Firefox looks good. The chromium-based browsers and Safari have this problem.

## Matrix

### OS

- macOS: Catalina 10.15.7
- macOS: Big Sur 11.6

### Browser

- Google Chrome: Version 94.0.4606.81 (Official Build) (x86_64)
- Microsoft Edge: Version 94.0.992.50 (Official build) (x86_64)
- Firefox: 93.0
- Safari: Version 15.0 (15612.1.29.41.4, 15612)

### IME

- ATOK for Mac Ver.32.1.0

**Comment by mariorod** [comment]
Thanks for reporting ... we will take a look this week and see what is the best way to address it.

**Reply by mesqueeb** [reply]
Keycode 13

**Reply by mesqueeb** [reply]
@mariorod Any progress on this? It's holding us back from using the new projects because we can't type properly : )

**Reply by mariorod** [reply]
@mesqueeb We are working on a PR and we hope to be able to push to production a fix this week

**Comment by azenMatt** [comment]
Hi everyone - this should now be fixed, can you please take a look and let us know if you are still having any problems? Thank you!

**Reply by peaceiris** [reply]
Thank you! LGTM ;)

**Reply by mesqueeb** [reply]
@azenMatt 
For me it works only on Google Chrome. Not on Safari.

Safari Version 15.1
and I'm using Google Japanese input: https://www.google.co.jp/ime/

**Reply by azenMatt** [reply]
Sorry about that @mesqueeb - we will take another look at this

**Reply by peaceiris** [reply]
This problem appears again to me when I select a suggestion by ATOK IME with the tab key.

**Reply by hussam-i-am** [reply]
@mesqueeb This should now be fixed for `Google Japanese input`. @peaceiris I was not able to confirm if it is still happening ATOK IME, can you take a look and let us know? Thanks!

**Reply by peaceiris** [reply]
@hussam-i-am I still have the problem using ATOK.

**Comment by sainu** [comment]
I am facing this on another UI.

In the title input on the drawer screen of the issue details, the title is saved each time a character is selected from the conversion table.

This makes it impossible to smoothly input titles in Japanese.

I hope to fix it because the github projects beta is great! Thank you!

https://user-images.githubusercontent.com/12888685/173263080-e2f7e9ce-70ea-4b99-ad14-1728d5a0a272.mov

**Reply by dmarcey** [reply]
Thanks for reporting @sainu, sorry about the frustrating experience here! We've identified a fix on our end and hope to roll it out this week - I'll update this thread when it is available.

**Reply by dmarcey** [reply]
@sainu - I believe that this fix has now rolled out - please let us know if you continue to see the issue!

**Reply by sainu** [reply]
@dmarcey I just confirmed it's fixed! Thanks for the fix.

**Reply by khunyshnxskul00** [reply]
https://www.mi.com/global/support/contact

**Reply by khunyshnxskul00** [reply]
https://github.com/googleapis/python-translate.git

**Comment by j0siepy** [comment, marked as answer]
Hello! Closing this out as it looks like a fix has been shipped!
